sdk/devcenter/Azure.Developer.DevCenter/src/autorest.md

```diff
@@ -7,18 +7,138 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-  - https://github.com/Azure/azure-rest-api-specs/blob/af3f7994582c0cbd61a48b636907ad2ac95d332c/specification/devcenter/data-plane/Microsoft.DevCenter/preview/2022-11-11-preview/devcenter.json
-  - https://github.com/Azure/azure-rest-api-specs/blob/af3f7994582c0cbd61a48b636907ad2ac95d332c/specification/devcenter/data-plane/Microsoft.DevCenter/preview/2022-11-11-preview/devbox.json
-  - https://github.com/Azure/azure-rest-api-specs/blob/af3f7994582c0cbd61a48b636907ad2ac95d332c/specification/devcenter/data-plane/Microsoft.DevCenter/preview/2022-11-11-preview/environments.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/552eaca3fb1940d5ec303746017d1764861031e6/specification/devcenter/data-plane/Microsoft.DevCenter/stable/2023-04-01/devcenter.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/552eaca3fb1940d5ec303746017d1764861031e6/specification/devcenter/data-plane/Microsoft.DevCenter/stable/2023-04-01/devbox.json
+  - https://github.com/Azure/azure-rest-api-specs/blob/552eaca3fb1940d5ec303746017d1764861031e6/specification/devcenter/data-plane/Microsoft.DevCenter/stable/2023-04-01/environments.json
 
 namespace: Azure.Developer.DevCenter
 security: AADToken
 security-scopes: https://devcenter.azure.com/.default
 
 directive:
+  # Move project name to method level parameters
+  - from: swagger-document
+    where: $.parameters["ProjectNameParameter"]
+    transform: >-
+      $["x-ms-parameter-location"] = "method"
+
   # Ensure we use Uri rather than string in .NET
   - from: swagger-document
     where: "$.parameters.EndpointParameter"
     transform: >-
       $["format"] = "url";
+
+  # @autorest/csharp hasn't yet shipped a version which understands operation-location. Until then, pull the options.
+  # TODO: remove these directives once fix makes it to a version of @autorest/csharp
+  - from: swagger-document
+    where-operation: DevBoxes_DeleteDevBox
+    transform: >-
+      delete $["x-ms-long-running-operation-options"];
```

Comment:
Given the SDK is prepped for GA, these changes should be done in the swagger file so each language can benefit from it. Do you have the swagger PR already in the works?

Reply:
Yes, swagger PR for the GA release has been merged a while ago.

Reply:
If the swagger PR incorporating these changes is merged, can we proceed with removing these transforms from the autorest.md?

Reply:
The changes have been made in the newer version of the API:
2023-07-01-preview

The transforms are still required for 2023-04-01